src/Xrefcheck/Scanners/Markdown.hs

```diff
@@ -388,23 +459,30 @@ getXrefcheckContent node =
 -- As our annotations are always oneliners, we can fix this by simply setting
 -- end line equals to start line and calculating end column from start column
 -- and annotation length.
-getPosition :: Node -> Maybe PosInfo
-getPosition node@(Node pos _ _) = do
+getPosition :: C.Node -> Maybe PosInfo
+getPosition node@(C.Node pos _ _) = do
   annLength <- length . T.strip <$> getHTMLText node
   PosInfo sl sc _ _ <- pos
   pure $ PosInfo sl sc sl (sc + annLength - 1)
 
 -- | Extract `IgnoreMode` if current node is xrefcheck annotation.
```

Comment:
I think that this comment should be updated:

`IgnoreMode` `->` `GetAnnotation`